src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -185,7 +186,7 @@ private function getCriteria(Request $request, MapEntity $options, ObjectManager
 
             return $criteria;
         } elseif (null === $mapping) {
-            trigger_deprecation('symfony/doctrine-bridge', '7.1', 'Relying on auto-mapping for Doctrine entities is deprecated for argument $%s of "%s": declare the identifier using either the #[MapEntity] attribute or mapped route parameters.', $argument->getName(), method_exists($argument, 'getControllerName') ? $argument->getControllerName() : 'n/a');
+            trigger_deprecation('symfony/doctrine-bridge', '7.1', 'Relying on auto-mapping for Doctrine entities is deprecated for argument $%s of "%s": declare it using either the #[MapEntity] attribute or mapped route parameters.', $argument->getName(), method_exists($argument, 'getControllerName') ? $argument->getControllerName() : 'n/a');
```

Comment:
maybe "one" instead of "it" :thinking:  (on both messages)
this looks better to me now that I read the message again